tools/sz_repo_cli/lib/src/commands/src/deploy_web_app_command.dart

```diff
@@ -192,27 +190,3 @@ class DeployWebAppCommand extends CommandBase {
     return currentCommit;
   }
 }
-
-class _WebAppConfig {
```

Comment:
I remove the `_WebConfigConfig` class since it was too overkill, `_webAppConfigs` wasn’t suitable to support multiple projects with the same stage and `_WebConfigConfig` would duplicate the Firebase Hosting targets.